src/gp/src/GPMSA.C

```diff
@@ -968,6 +972,9 @@ GPMSAFactory<V, M>::setUpEmulator()
     (*BT_Wy_B_inv)(i,i) +=
       this->m_opts->m_observationalCovarianceRidge;
 
+  // Emulator expects the error matrix, so invert Wy to get it.
+  m_observationErrorMatrix.reset(new M(Wy.inverse()));
+
```

Comment:
This is a block diagonal matrix where the blocks are what the user provided.  Why are we first inverting the blocks, then making that big block diagonal matrix, then inverting the big matrix?